bom/runtime/pom.xml

```diff
@@ -517,6 +517,12 @@
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-smallrye-openapi</artifactId>
                 <version>${project.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.jboss.shrinkwrap</groupId>
+                        <artifactId>shrinkwrap-api</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
Wondering if it's an issue in `smallrye-openapi` itself and if the dependency should be test there?

Reply:
ShrinkWrap is used by this class in SmallRye OpenAPI: https://github.com/smallrye/smallrye-open-api/blob/master/implementation/src/main/java/io/smallrye/openapi/api/util/ArchiveUtil.

Reply:
I wonder if this should be a test class or if it's really supposed to be used in user applications.

@EricWittmann ?

Reply:
See my comment in this PR.